src/bin/report_collector.rs

```diff
@@ -144,6 +147,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
                 IpaSecurityModel::Malicious,
                 config,
                 &clients,
+                false,
```

Comment:
I'd recommend a new Enum here instead of a boolean. That way, you don't need to look up the method signature to know what it means. Something like `IpaQueryStyle::OPRF`